### ISSUE TITLE
Click 8.0 and pywbemcli have incompatibility because of click-repl

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbemtools#819]
Click completely rewrote there autocompletion and in the process deleted the method _bashcomplete and its corresponding model that is used by click-repl which we use.  

This results in a failure when pywbemcli with the current alpha (master branch) of click 8 fails immediately upon startup because the autocomplete has been completely rewritten and the method _bashcomplete no longer exists in click

This means that we cannot move to click 8.0 when it is released unless clic_repl is fixed.

The following pr on click repl has been filed : https://github.com/click-contrib/click-repl/issues/72

**Comment by andy-maier** [comment]
Mitigated in PR #816.

**Comment by andy-maier** [comment]
**DISCUSSION**: Final solution?

**Comment by andy-maier** [comment]
Removing the pinning of Click to <8.0 is possible only on Python 3.4 and higher, and only once click-repl supports the new autocompletion API of Click 8.0.

**Comment by andy-maier** [comment]
Karl will create another issue for the click 8 usage - this one is marked as resolved.
